contracts/EpochTokenLocker.sol

```diff
@@ -124,9 +128,14 @@ contract EpochTokenLocker {
     /**
      * internal functions
      */
+    function hasValidWithdrawRequest(address user, address token) internal view returns(bool) {
```

Comment:
Does it maybe make sense to expose this publicly since it's just a view function anyways?